src/site/content/en/blog/ta-strategies/index.md

```diff
@@ -0,0 +1,266 @@
+---
+layout: post
+title: Pyramid or Crab? Find a testing strategy that fits 
+date: 2023-07-26
+subhead: >
+  Discover how to combine different testing types into a reasonable strategy that matches your project.
+description: >
+  Discover how to combine different testing types into a reasonable strategy that matches your project.
+authors:
+  - ramona
+hero: image/dPDCek3EhZgLQPGtEG3y0fTn4v82/CS0pyEGhIFszXNpM6Ndv.jpg
+alt: A lot of test tubes to depict testing.
+tags:
+  - blog
+  - test-automation
+  - testing
+---
+
+Welcome back! In the [last article](/ta-types), we laid down lots of groundwork about how to approach the different testing types and what they contain, and we clarified the testing type definitions. Remember our little meme image from before? We wondered how all those testing types we learned about could work together.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/n3LPuwApNcCRT5S3zaoq.jpeg", alt="ALT_TEXT_HERE", width="800", height="450" %}
+
+So next up, we will learn exactly that. This article will give an introduction on how to combine these testing types into a reasonable strategy that matches your project.
+
+I want to share some recommendations on which testing strategies might fit which size of project. Let’s begin with a short overview—an attempt to generalize based on my own experience, so it can naturally deviate from your use case. I hope this is a good starting point:
```

Comment:
```suggestion
I want to share some recommendations on which testing strategies might fit your project, depending on its size. Let’s begin with a short overview—an attempt to generalize based on my own experience, so it can naturally deviate from your use case. I hope this is a good starting point:
```